src/helpers/datasetDetails.js

```diff
@@ -6,17 +8,67 @@ export function getDatasetDetails(state) {
   datasetDetails.description = getDatasetDescription(state);
   datasetDetails.numRows = state.data.length;
   datasetDetails.isUserUploaded = isUserUploadedDataset(state);
+  datasetDetails.potentialUses = getPotentialUses(state);
+  datasetDetails.potentialMisuses = getPotentialMisuses(state);
   return datasetDetails;
 }
 
+function getPotentialUses(state) {
+  if (
+    state.metadata &&
+    state.metadata.name &&
+    state.metadata.card &&
+    state.metadata.card.context &&
+    state.metadata.card.context.potentialUses
+  ) {
+    const datasetId = state.metadata.name;
+    const fallback = state.metadata.card.context.potentialUses;
+    return I18n.t("potentialUses",
+      {
+        scope: ["datasets", datasetId, "card", "context"],
+        default: fallback
+      }
+    );
+  }
+  return undefined;
+}
+
+function getPotentialMisuses(state) {
```

Comment:
There is some potential DRY-ness available with a small change to merge these two functions into one to work for both `potentialMisuses` and `potentialUses`, if you'd like.